lib/internal/Magento/Framework/Module/Test/Unit/ModuleList/SorterTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Magento\Framework\Module\Test\Unit\ModuleList;
+
+use Magento\Framework\Module\ModuleList\Sorter;
+use PHPUnit\Framework\TestCase;
+
+class SorterTest extends TestCase
```

Comment:
Please make sure new class is 100% covered. At least there is no test for `Circular sequence reference from` exception currently.